include/ocpp/common/message_queue.hpp

```diff
@@ -579,18 +596,27 @@ template <typename M> class MessageQueue {
     void pause() {
         EVLOG_debug << "pause()";
         std::lock_guard<std::mutex> lk(this->message_mutex);
+        this->pause_resume_ctr++;
+        this->resume_timer.stop();
         this->paused = true;
         this->cv.notify_one();
         EVLOG_debug << "pause() notified message queue";
     }
 
     /// \brief Resumes the message queue
-    void resume() {
-        EVLOG_debug << "resume()";
+    void resume(unsigned int delay_seconds = 0) {
```

Comment:
Suggestion: Use chrono type here so it is perfectly clear what the unit is

Reply:
Done